addons/l10n_pe/models/res_bank.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class ResBank(models.Model):
+    _inherit = "res.bank"
+
+    def _get_fiscal_country_codes(self):
+        return ','.join(self.env.companies.mapped('account_fiscal_country_id.code'))
+
+    l10n_pe_edi_code = fields.Char(
+        "Code (PE)",
+        help="Bank code assigned by the SUNAT to identify banking institutions.")
+    fiscal_country_codes = fields.Char(store=False, default=_get_fiscal_country_codes)
```

Comment:
l10n_hk something uses the country of the bank with a related country_code as country.code.  https://github.com/odoo/enterprise/blob/9f4dc58dc6adbed1d939328ef8136536978be940/l10n_hk_hr_payroll/models/res_bank.py#L12